From 8f406326af1cd8a6286953fb1ad9108b5d048a77 Mon Sep 17 00:00:00 2001
From: Yu Liao <liaoyu15@huawei.com>
Date: Sat, 29 Jan 2022 00:22:56 +0800
Subject: [PATCH] kabi: Reserve space for struct acpi_device_power
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 8f406326af1cd8a6286953fb1ad9108b5d048a77
Modified-by-SEL: No


hulk inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I4QQWY
CVE: NA

--------------------------

Reserve space for struct acpi_device_power

Signed-off-by: Yu Liao <liaoyu15@huawei.com>
Reviewed-by: Xiongfeng Wang <wangxiongfeng2@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 include/acpi/acpi_bus.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/acpi/acpi_bus.h b/include/acpi/acpi_bus.h
index 6ad3b89a8a2e..a072afc7609a 100644
--- a/include/acpi/acpi_bus.h
+++ b/include/acpi/acpi_bus.h
@@ -11,6 +11,7 @@
 
 #include <linux/device.h>
 #include <linux/property.h>
+#include <linux/kabi.h>
 
 /* TBD: Make dynamic */
 #define ACPI_MAX_HANDLES	10
@@ -277,6 +278,8 @@ struct acpi_device_power {
 	int state;		/* Current state */
 	struct acpi_device_power_flags flags;
 	struct acpi_device_power_state states[ACPI_D_STATE_COUNT];	/* Power states (D0-D3Cold) */
+
+	KABI_RESERVE(1)
 };
 
 /* Performance Management */
-- 
2.26.2

